modules/Profile/components/ResetPasswordFormModal/ResetPasswordFormModal.helpers.ts

```diff
@@ -0,0 +1,17 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+export const resetPasswordSchema = z
+  .object({
+    oldPassword: z.string().min(1, { message: "Old password is required" }),
+    newPassword: z.string().min(8, { message: "Password must be at least 8 characters long" }),
```

Comment:
Question: password must be 8 characters long is this the only validation that we have for passwords?

Reply:
Thank you for pointing this. I will give more validation, this was done because of testing purpose.